prisma/migrations/20220217215935_updated_schema_graph_ql/migration.sql

```diff
@@ -0,0 +1,136 @@
+  - You are about to drop the column `LocationID` on the `Comment` table. All the data in the column will be lost.
+  - You are about to drop the column `UserID` on the `Comment` table. All the data in the column will be lost.
+  - You are about to drop the column `BusinessID` on the `Location` table. All the data in the column will be lost.
+  - You are about to drop the column `Password` on the `User` table. All the data in the column will be lost.
+  - The primary key for the `UserReview` table will be changed. If it partially fails, the table could be left without primary key constraint.
+  - You are about to drop the column `BusinessID` on the `UserReview` table. All the data in the column will be lost.
+  - You are about to drop the column `ReviewID` on the `UserReview` table. All the data in the column will be lost.
+  - You are about to drop the column `UserID` on the `UserReview` table. All the data in the column will be lost.
+  - A unique constraint covering the columns `[BusinessID]` on the table `Business` will be added. If there are existing duplicate values, this will fail.
+  - A unique constraint covering the columns `[name]` on the table `Business` will be added. If there are existing duplicate values, this will fail.
+  - A unique constraint covering the columns `[locationId]` on the table `Business` will be added. If there are existing duplicate values, this will fail.
+  - A unique constraint covering the columns `[businessId,businessName]` on the table `Location` will be added. If there are existing duplicate values, this will fail.
+  - Added the required column `createdAt` to the `Business` table without a default value. This is not possible if the table is not empty.
+  - Added the required column `updatedAt` to the `Business` table without a default value. This is not possible if the table is not empty.
+  - Added the required column `userId` to the `Business` table without a default value. This is not possible if the table is not empty.
+  - Added the required column `locationId` to the `Comment` table without a default value. This is not possible if the table is not empty.
+  - Added the required column `userId` to the `Comment` table without a default value. This is not possible if the table is not empty.
+  - Added the required column `createdAt` to the `Location` table without a default value. This is not possible if the table is not empty.
+  - Added the required column `updatedAt` to the `Location` table without a default value. This is not possible if the table is not empty.
+  - Added the required column `password` to the `User` table without a default value. This is not possible if the table is not empty.
+  - Added the required column `reputation` to the `UserReview` table without a default value. This is not possible if the table is not empty.
+  - Added the required column `userId` to the `UserReview` table without a default value. This is not possible if the table is not empty.
+
+*/
```

Comment:
This is a lot of warnings - are you sure they aren't an issue?

Reply:
The warnings mention the changes that were made to the database